.github/workflows/ci.yml

```diff
@@ -50,41 +60,62 @@ jobs:
       - name: Run build
         run: npm run build
 
-      - name: Install yalc
-        run: npm install -g yalc
-
       - name: Yalc publish
-        run: yalc publish
+        run: npx yalc publish
 
       - name: Delete top level node_modules
         run: rm -rf node_modules
 
       - name: Test CJS imports
         working-directory: e2e/test-import-cjs
-        run: yalc add @autoblocks/client && npm install && npm run start
+        run: npx yalc add @autoblocks/client && npm install && npm run start
         env:
           AUTOBLOCKS_INGESTION_KEY: test
 
       - name: Test ESM imports
         working-directory: e2e/test-import-esm
-        run: yalc add @autoblocks/client && npm install && npm run start
+        run: npx yalc add @autoblocks/client && npm install && npm run start
         env:
           AUTOBLOCKS_INGESTION_KEY: test
 
       - name: Test ESM imports & API client
         working-directory: e2e/test-api-client-esm
-        run: yalc add @autoblocks/client && npm install && npm run start
+        run: npx yalc add @autoblocks/client && npm install && npm run start
         env:
           AUTOBLOCKS_API_KEY: ${{ secrets.CI_AUTOBLOCKS_API_KEY }}
           AUTOBLOCKS_INGESTION_KEY: ${{ secrets.CI_AUTOBLOCKS_INGESTION_KEY }}
 
       - name: Test Prompt Manager
         working-directory: e2e/test-prompt-manager-ts
         run: |
-          yalc add @autoblocks/client && \
+          npx yalc add @autoblocks/client && \
           npm install && \
           npm run gen && \
           tsc --noEmit && \
           npm run test
         env:
+          # Org-scoped API key
           AUTOBLOCKS_API_KEY: ${{ secrets.CI_AUTOBLOCKS_API_KEY }}
+          # User-scoped API key
+          AUTOBLOCKS_API_KEY_USER: ${{ secrets.CI_AUTOBLOCKS_API_KEY_USER_NICOLE }}
```

Comment:
i added my own user-scoped API key to the secrets on github actions but when running e2e tests locally you can set `AUTOBLOCKS_API_KEY_USER` to your own user-scoped API key b/c i made the revision used in the "Pinned Undeployed" test a shared revision

<img width="960" alt="Screenshot 2024-04-28 at 11 32 10 AM" src="https://github.com/autoblocksai/javascript-sdk/assets/7498009/de524d56-27cb-4431-be6b-61f5157609af">

Reply:
alternatively i could run the undeployed e2e tests in a separate step so that it can just be called `AUTOBLOCKS_API_KEY` everywhere